qutebrowser/browser/downloads.py

```diff
@@ -373,7 +392,6 @@ def delete(self):
         except OSError:
             log.downloads.exception("Failed to remove partial file")
 
-    @pyqtSlot()
```

Comment:
Something must have gone wrong when I did the rebase. I didn't mean to remove the decorator from either delete or retry methods.